src/olympia/addons/tests/test_utils_.py

```diff
@@ -46,11 +45,11 @@
     ('Firefox add-on for Firefox', False, 'foo@bar.com', False),
     ('Firefox add-on for Firefox', True, 'foo@mozilla.com', False),
     ('Foobarfor Firefox', False, 'foo@bar.com', False),
-    ('Better Privacy for Firefox!', True, 'foo@bar.com', False),
+    ('Better Privacy for Firefox!', False, 'foo@bar.com', False),
     ('Firefox awesome for Mozilla', False, 'foo@bar.com', False),
     ('Firefox awesome for Mozilla', True, 'foo@mozilla.org', False),
 
-    # And with the content optimization waffle onL
+    # And with the content optimization waffle on
```

Comment:
Is this comment still accurate?

Reply:
Yes, I was just fixing the typo, presumably.

Reply:
I dont see the waffle being used in utils.py?

Reply:
I didn't want to change the tests too much, assuming we would have a cleanup one that waffle has been in prod for a while and gets removed. If you prefer, I can clean up this test.

Reply:
It would be nice to keep the test in sync with the function.

This comment suggests that we're using a waffle in the function but that doesn't seem to be the case.

Reply:
67c2d08152